backend/forecastbox/api/routers/product.py

```diff
@@ -59,38 +63,56 @@ def select_from_params(available_spec: Qube, params: dict[str, Any]) -> Qube:
 
     return available_spec
 
+def _sort_values(values: list[Any]) -> list[Any]:
+    if all(str(x).isdigit() for x in values):
+        values = list(map(float, sorted(values, key=float)))
+        return values
+    values = list(sorted(values, key=lambda x: str(x).lower()))
+    return values
```

Comment:
puzzles me a bit -- this sort a numerical list (say ["12", "9"]) in numerical order (["9", "12"]), but if its mixed (["12", "9", "hello"]) all of sudden its reversed (ie "12" will come before "9")

dont you want some sort of https://stackoverflow.com/questions/2669059/how-to-sort-alpha-numeric-set-in-python to make it stable? The first solution there is nice